.github/workflows/reusable_lint.yml

```diff
@@ -46,9 +46,13 @@ jobs:
           extensions: ${{ inputs.extensions }}
           clangFormatVersion: ${{ inputs.version }}
 
-      - name: Lint check for Rust app
+      - name: Install rust for Rust app
         if: ${{ needs.call_get_app_metadata.outputs.is_rust == 'true'}}
-        uses: actions-rs/cargo@v1
+        uses: actions-rs/toolchain@v1
```

Comment:
As the CI is running in our Docker builder (`ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest`), I think that using a github action to get Rust is useless.

Reply:
@yogh333 I set it as you were using the GitHub Action for cargo. Shouldn't it be necessary either?

Reply:
Yes it might be the good opportunity to remove the github action for cargo, as highlighted in #83.
Thanks !

Reply:
@yogh333 done!